cmd/controller-bootstrap/command/apiutil.go

```diff
@@ -0,0 +1,19 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//     http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package command
+
+// todo: getServiceResources infers aws-sdk-go to fetch the service metadata and custom resource names
```

Comment:
super nit: if you make it "TODO" instead of "todo" then the IDE will highlight all the TODOs for you to address

Reply:
Great!